README.md

```diff
@@ -589,6 +590,29 @@ Parameters:
         -   `ret` is the result
         -   If no callback is provided, error will be shown as a VSCode notification.
 
+### vscode.into_insert(callback)
```

Comment:
`with_` is probably more conventional than `into_`.

```suggestion
### vscode.with_insert(callback)
```

Reply:
I wanted to use "with" at first, but considering that this function can be used as an ordinary function call without providing a callback function. "With" is not appropriate?